gh-72719: Remove deprecated `asyncore` and `asynchat` modules (#96580)
python/cpython: Misc/NEWS.d/next/Library/2022-09-05-17-08-56.gh-issue-72719.jUpzF3.rst

Remove modules :mod:`asyncore` and :mod:`asynchat`, which were deprecated by :pep:`594`.

> They are temporarily moved to :mod:`test.support` so some other tests that rely on them keep working.

Reviewer: I suggest to remove the second sentence. I prefer to hide them as well as possible :-)

Author: Done, thank you for the review!

> Remove deprecated as per :pep:`594` modules :mod:`asyncore` and :mod:`asynchat`.
> 
> They are moved to ``test.support`` so, some other tests that rely on them keep
> working.

Reviewer 2: ```suggestion
Remove modules :mod:`asyncore` and :mod:`asynchat`, which were deprecated by :pep:`594`.

They are temporarily moved to :mod:`test.support` so some other tests that rely on them keep working.
```
